### PR TITLE
Fix linting in script blocks in vue files

### PR DESCRIPTION
**Pull request by derick-montague**
- Removed the .eslintrc and added the pretteri rules to the eslintrc file
- Ran `npm run lint --fix`

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -6,7 +6,13 @@ module.exports = {
   extends: ['plugin:vue/essential', '@vue/prettier'],
   rules: {
     'no-console': 'off',
-    'no-debugger': process.env.NODE_ENV === 'production' ? 'error' : 'off'
+    'no-debugger': process.env.NODE_ENV === 'production' ? 'error' : 'off',
+    'prettier/prettier': [
+      'error',
+      {
+        singleQuote: true
```
Comment (gtmills) [inline .eslintrc.js:13]: When we push upstream we should add comments for *why* we are doing things like this.

**Review by gtmills — APPROVED** [review]

File: .eslintrc.js (hunk 1 above)
Comment (derick-montague) [reply .eslintrc.js:13]: I agree and also feel we should have documentation for all the best practices and for the most part this is stylistic in nature. However, many JS libraries and Node uses this convention which has made it natural for JavaScript developers. Most important part to me is consistency and we were using single quotes in the the AngularJS code base primarily.

Comment (gtmills) [reply .eslintrc.js:13]: agreed.

**Review by yoshiemuranaka — APPROVED** [review]

**Comment by derick-montague** [comment]
Migrated to webui-vue in https://gerrit.openbmc-project.xyz/c/openbmc/webui-vue/+/28732

**Pull request merged**